libbeat/processors/add_host_metadata/add_host_metadata.go

```diff
@@ -178,7 +179,10 @@ func (p *addHostMetadata) loadData(checkCache bool, useFQDN bool) error {
 
 	hostname := h.Info().Hostname
 	if useFQDN {
-		fqdn, err := h.FQDN()
+		ctx, cancel := context.WithTimeout(context.Background(), 1*time.Minute)
+		defer cancel()
+
+		fqdn, err := h.FQDN(ctx)
```

Comment:
I thought you went with `FQDNWithContext(ctx)`.

Reply:
Yes, I forgot to update this PR after that 🤦.  Thanks for catching.

Reply:
Fixed in febb1331a0ac07a6bfec346362af80fc2dc1140e.